src/main/java/com/monitorjbl/xlsx/impl/StreamingWorkbookReader.java

```diff
@@ -188,19 +188,23 @@ void lookupSheetNames(XSSFReader reader) throws IOException, InvalidFormatExcept
   }
 
   @Override
-  public void close() {
+  public void close() throws IOException {
     try {
       for(StreamingSheet sheet : sheets) {
         sheet.getReader().close();
       }
       pkg.revert();
     } finally {
       if(tmp != null) {
-        log.debug("Deleting tmp file [" + tmp.getAbsolutePath() + "]");
+        if (log.isDebugEnabled()) {
```

Comment:
This seems a tad unnecessary, I'm pretty sure the logging framework will do this with just a call to `log.debug()`

Reply:
avoids the overhead of doing the string concat if debug is not enabled but not a big deal either way

Reply:
That's a good reason to leave it as-is. I honestly just need to do a pass through the code at some point to use the formatted string form of the logger methods. This will effectively skip string concatenation if debugging is not enabled:

```
log.debug("Deleting tmp file [{}]",  tmp.getAbsolutePath())
```

I think I just didn't know about that when I initially wrote this code :(